instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/sessionreplay/IntExt.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sdk.integration.sessionreplay
+
+/**
+ * Normalizes an Int value (font size, view dimension, view position, etc.) according with the
+ * device pixels density.
+ * Example: if a device has a DPI = 2, the normalized height of a view will be
+ * view.height/2.
+ * @param density
+ */
+internal fun Int.densityNormalized(density: Float): Int {
```

Comment:
maybe instead of 2 classes `IntExt` and `LongExt` we can have single `NumberExt`?

Reply:
I think Kotlin provides dedicated classes / types when it comes to extensions right ?